region/norfair/east.json

```diff
@@ -3840,9 +3849,14 @@
                     "heatProof",
                     "canHeatRun"
                   ]},
-                  {"heatFrames": 90}
+                  {"heatFrames": 90},
+                  {"resetRoom":{
+                    "nodes": [2],
+                    "mustStayPut": true
+                  }}
                 ],
-                "openEnd": 1
+                "openEnd": 1,
+                "note": "Can't charge in lava, so doesn't work coming in from the top left door"
```

Comment:
Unless you have Gravity

Reply:
No, you can't charge in lava. Gravity allows you to charge in acid.

Reply:
Weird TIL